rudder-jetty/SOURCES/Makefile

```diff
@@ -70,6 +70,7 @@ localdepends: ./jetty7
 
 	cp ./jetty7/bin/jetty-rpm.sh ./jetty7/bin/jetty-sles.sh
 	patch -p0 -s < patches/jetty/jetty-init-sles.patch
+	patch -p0 -s < patches/jetty/jetty-init-ulimit.patch
```

Comment:
The ticket says it was patched in /opt/rudder./... whcih is the source of what is copied to /etc/init.d/rudder-jetty, so I think this patch will be enough actually.